app/views/patients/_patient_dashboard.html.erb

```diff
@@ -1,39 +1,42 @@
 <div id="patient_dashboard_content">
```

Comment:
the diff on this looks big, but it's basically changing col values and putting things in two rows (and spacing out the form fields).

Reply:
github's ignore whitespace to the rescue!